templates/tensorflow/r1.15/base.libsonnet

```diff
@@ -0,0 +1,14 @@
+local base = import "../base.libsonnet";
```

Comment:
Remember to add a copyright header to new files.

Reply:
Done